pkg/detectors/saladcloudapikey/saladcloudapikey.go

```diff
@@ -0,0 +1,98 @@
+package saladcloudapikey
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	apiKey        = regexp.MustCompile(`\b(salad_cloud_[0-9A-Za-z]{1,7}_[0-9A-Za-z]{7,235})\b`)
```

Comment:
@seniorquico Is there any reason to select the range of `{7,235}`? I generated three to four times the API Access key, All of the time, the last part, I got of length 49 characters.

Reply:
We are currently wrapping up our GitHub Secret Scanning integration. We spec'd out the API keys similar to GitHub's based on this [blog post](https://github.blog/engineering/behind-githubs-new-authentication-token-formats/). The max 255 character length was just to keep options open for future scenarios with higher entropy keys.

I don't think it's a problem to shorten this if you think that's a better idea for the near-term. Please advise, and I can update the PR if necessary.